docs/dev-notes/DEVELOPMENT.md

```diff
@@ -0,0 +1,135 @@
+
+> **Warning**
+> 
+> **Please make sure you are at root directory of the project before running commands mentioned below**
+
+### Connect the app to a workspace in Customer.io
+
+- Make a copy of `env.sample.js` by running `cp env.sample.js env.js` at project root directory
+- Update `env.js` with your `siteId` and `apiKey`
+
+### Install Dependencies
+
+Before running any app, make sure the dependencies are installed by running following command:
+
+```
+yarn install
+```
+
+In case you add/update dependencies later, you will need to run this command again to install the updated dependencies.
+
+### Android
+
+*Note: It's assumed that you have Android Studio installed on your machine before continuing*
+
+This project requires Firebase setup in order to build the app. You can quickly get the app to build with following command
+
+```
+cp android/app/google-services.json.example android/app/google-services.json
+```
+
+But if you want to do anything like push notifications, you will want to follow [Firebase setup](https://firebase.google.com/docs/android/setup) to add a new Android app to your Firebase project. This will give you a `google-services.json` file to download. Save this file at `android/app/google-services`. 
+
+#### Option 1: Using Command Line
+
+Use the following commands to run the app
+
+```
+npx react-native run-android
+```
+
+Android app will be launched on physical device if connected using `adb`, if not, it should open an emulator and run the app on it. However, if you have multiple devices connected, you can run on a specific device by running following command:
+
+```
+npx react-native run-android --deviceId=DEVICE_ID
+```
+
+Where `DEVICE_ID` can be obtained by running `adb devices` command.
+
+#### Option 2: Using Android Studio
+
+- Open the `android` folder in Android Studio
+- Build the app in Android Studio
+- Run the app
+
+### iOS
+
+*Note: It's assumed that you have XCode installed on your machine before continuing*
+
+iOS app requires installing libraries using CocoaPods. You can install it by running following commands
+
+```
+cd ios
+pod install
+cd ..
+```
+
+Next, follow these steps:
+
+- Jump into `ios` folder and open [`.xcworkspace`](/ios/SampleApp.xcworkspace) file to setup the project
+- [Configure iOS Code Signing](https://github.com/customerio/mobile/blob/main/ios_code_signing.md#how-do-i-setup-my-computer-for-code-signing) to download the provisioning profiles and certificates for development to your machine. 
+
+> **Note:**
+> Make sure that the deployment target is set to the minimum [required by the iOS SDK](https://github.com/customerio/customerio-ios#readme).
+
+#### Option 1: Using Command Line
+
+**To run the app on iOS Simulator**
+
+Use the following commands to run the app on iOS Simulator
+
+```
+npx react-native run-ios
+```
+**To run the app on device**
+
+To run using terminal
+
+```
+npx react-native run-ios --device "Your iPhone"
```

Comment:
Can we include instructions for how to find the name of iPhone? The "Your iPhone" *value*?

Reply:
I'm not really sure how to find it. I generally look in XCode, not sure if it is the best/easiest way? I'll take a look at it. Meanwhile, if you have any suggestions, please let me know.

Reply:
Added a command that works fine for me, please feel free to update if it isn't the correct recommendation 😅